sml_small/editing/totals_and_components/totals_and_components.py

```diff
@@ -392,8 +394,21 @@ def totals_and_components(
                             total=input_parameters[InputParameters.TOTAL.value],
                         )
 
-        # Return the raw string instead of the enum value
-        output_list["tcc_marker"] = output_list["tcc_marker"].value
+        final_components = [
+            str(component) for component in output_list["final_components"]
+        ]
+
+        # Return the values as raw strings instead of decimal
+        output_list = {
+            "identifier": identifier,
+            "absolute_difference": str(output_list["absolute_difference"]),
```

Comment:
It may be better to convert to string if type is decimal/ if not None, it outputting "None" would cause some really weird situations if actually using the method in a non-unit-testing scenario where we'd then have to check the data afterwards and potentially fix them

Reply:
Ok no worries will do that now